docs/source/devguides/supporting/examples/MatLabAgent.rst

```diff
@@ -0,0 +1,222 @@
+
+
+
+Setup on Windows
+----------------
+
+Install pre-requisites
+~~~~~~~~~~~~~~~~~~~~~~~
+1. Install python 2.7 from `here <https://www.python.org/downloads/windows/>`__.
+
+2. Install MatLab engine from  `here <https://www.mathworks.com/help/matlab/matlab_external/install-the-matlab-engine-for-python.html>`_.
+
+Install StandAloneMatLab Agent
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The standalone MatLab agent is designed to be usable in a 
+Windows environment. 
+
+.. warning:: 
+
+        VOLTTRON is not designed to run in a Windows environment. 
+        Outside of cases where it is stated to be usable in a 
+        Windows environment, it should be assumed that it will
+        NOT function as expected.
+
+
+1. Download VOLTTRON
+
+   Download the VOLTTRON develop repository from github. Download the zip  
+   from `GitHub <https://github.com/VOLTTRON/volttron/tree/develop>`_ and unzip 
+   
+   |github-image|
+
+2. Setup the PYTHONPATH
+   
+   Open the Windows explorer, and navigate to "Edit environment variables for your account."
+   
+   |cmd-image|
+
+   Select "New"
+   
+   |env-vars-image_1|
+   
+   For "Variable name" enter: "PYTOHNPATH"
+   For "Variable value" either browse to your volttron installation, or enter in the path to your volttron installation.
+   
+   |env-vars-image_2|
+   
+   Select "OK" twice.
+
+3. Set up the environment.
+   
+   Open up the command prompt.
+   
+   |cmd-image_2|
+   
+   Naviage to your volttron installation.
+   
+   ``cd \Your\directory\path\to\volttron-develop``
+   
+   Use pip to install and setup dependencies.
+   
+   ``pip install -r requirements.txt``
+   
+   ``pip install -e .``
+   
+   .. note::
+   
+     If you get the error doing the second step because of an already installed volttron from a different directory, manually delete the volttron-egg.link file from your <python path>\Lib\site-pacakages directory ( for example, del  C:\Python27\lib\site-packages\volttron-egg.link ) and re run the second command
+
+4. Configure the agent
+
+The configurations for the standalone agent is in setting.py in the same directory as the standalone agent.
+
+1. settings.py
+   * 'vip_address' needs to be set to the address of your volttron instance
+   
+   * 'port' needs to be set to the port of your volttron instance
+   
+   * 'server_key' needs to be set to the public server key of your remote platform.
+     This can be obtained from the instance using the ``vctl auth serverkey`` command.
+   
+.. note::
+
+        It is recommended that you generate a new agent_public and agent_private
+        key for your standalone agent. This can be done using the ``vctl auth keypair``
+        command on your volttron instance. If you plan to use multiple standalone agents,
+        they will each need their own keypair.
+
+2. standalone_matlab.py
+
+   * @PubSub.subscribe('pubsub','matlab/to_agent/1') should be set to the 
+     topic you choose in your config file. (line 74)
```

Comment:
Refer to the installation step above - i.e. create  link to step 3 under Setup on linux